test/utils/crd/crd_util.go

```diff
@@ -47,9 +50,6 @@ type Option func(crd *apiextensionsv1.CustomResourceDefinition)
 
 // CreateMultiVersionTestCRD creates a new CRD specifically for the calling test.
 func CreateMultiVersionTestCRD(f *framework.Framework, group string, opts ...Option) (*TestCrd, error) {
-	suffix := framework.RandomSuffix()
```

Comment:
```
suffix := fmt.Sprintf("%s-%x", framework.RandomSuffix(), time.Now().Unix()) 

```

this will be more than enough

Reply:
If this retry logic is something we already do for namespaces, it seems fine use the same pattern for CRDs? Slight preference to just retry as that reduces flake chances to 0 rather than near 0 but will defer to @aojea

Reply:
fair